CMakeLists.txt

```diff
@@ -253,6 +253,7 @@ option(BUILD_PACKAGE      "Build package"           YES)
 option(BUILD_PRIVACYGUARD "Build Privacy Guard"     YES)
 option(BUILD_CDS          "Build CDS - Common Diagnostic Stack"     YES)
 option(BUILD_LIVEEVENTINSPECTOR   "Build Live Event Inspector"      YES)
+option(BUILD_SUBSTRATESIGNALS "Build Substrate Signals"     YES)
```

Comment:
Will leave it for you to decide if the `substrate` keyword is not supposed to be Microsoft internal naming, as this will will be exposed in the public repo? Else, you may want to change the files names to something more generic :)

Reply:
@microsoft/1ds-c-substratesignals what do you folks think?